apps/shared/interactive_curve_view_controller.cpp

```diff
@@ -13,10 +13,12 @@ InteractiveCurveViewController::InteractiveCurveViewController(Responder * paren
   m_modelVersion(modelVersion),
   m_previousModelsVersions(previousModelsVersions),
   m_rangeVersion(rangeVersion),
+  m_interactiveRange(interactiveRange),
```

Comment:
m_interactRange should be initialized right before m_rangeButton, as it is declared in the header file, to avoid compilation warnings.